tox.ini

```diff
@@ -5,7 +5,7 @@
 
 [tox]
 isolated_build = true
-envlist = clean, py{38,39,310,311,312}-base, py{38,39,310,311,312}-grpc, py39-base-nicaiu, py39-base-nicai_utf8, report, docs
+envlist = clean, py{38,39,310,311,312,313}-base, py{38,39,310,311,312,313}-grpc, py39-base-nicaiu, py39-base-nicai_utf8, report, docs
```

Comment:
FYI - Python 3.8 is end of life, which per [NI Driver ADE Support Policy](https://dev.azure.com/ni/DevCentral/_git/Policy?path=/product_and_technology/Driver-ADE-Support.md&_a=preview) means we can remove it. We shouldn't do it here, though. I'll create a new issue for this.

Reply:
#647

Reply:
Yeah, I think we should cherry-pick the 3.13 fix into releases/1.0 and remove 3.8 in 1.1.